README.md

```diff
@@ -42,7 +48,14 @@ Please report issues on [GitHub][issueslist].
 
 Contact the author, [Pierre Ganty][email].
 
+## Requeriments
+
+To use the scripts it's needed [mustache][mustachepage] to be installed.
```

Comment:
Fix english: "You need to install mustache to generate graphs" (or something like this)